### PR TITLE
RV-207: Update Card table, List table bulk selection behaviour to match updated OXD standard

### PR DESCRIPTION
**Pull request by ShachiniMekala**
## Checklist

- [x] Test Coverage is 100% for the newly added code
- [x] Storybook stories are added/updated for the changed areas
- [x] Components standards defined [in this document](https://docs.google.com/document/d/16_Nd3VxE_lTD9pVkONQ0egn7IiwyX1pVXZjzl-V4tU8/) are followed
- [x] Code is linted properly
- [x] Developer testing is done for the affected areas
- [ ] Package version updated (not applicable to ent branch)
- [x] Changelog.md updated on possible breaking (applicable to ent branch)

### CODE REVIEW

File: components/src/core/components/Input/checkbox-input.scss [hunk 1]
```diff
@@ -128,3 +127,7 @@
     align-self: center;
   }
 }
+
+span.oxd-checkbox-input[tooltip]::after {
+  max-width: 150px;
```
Comment (Super-Chama) [inline components/src/core/components/Input/checkbox-input.scss:132]: why this is needed?

File: storybook/stories/core/components/List/list-data.json [hunk 2]
```diff
@@ -212,11 +212,13 @@
...
         "downloadResumeSlot": true,
-        "downloadApplicationFormSlot": true,
+        "downloadApplicationFormSlot": true,  
         "compareSlot": true
       },
       "vacancy": null,
-      "vacancyId": null
+      "vacancyId": null,
+      "isSelectDisabled" : true,
+      "selectDisabledTooltip": "Not allowed to delete candidates who are in progress"
```
Comment (Super-Chama) [inline storybook/stories/core/components/List/list-data.json:221]: better add flow property example also

**Review by Super-Chama — COMMENTED** [review]
~LGTM~ Some issue founds in later review. please address

File: components/src/core/components/Input/checkbox-input.scss (hunk 1 above)
Comment (ShachiniMekala) [reply components/src/core/components/Input/checkbox-input.scss:132]: UI team wanted to decrease the width of the tooltip

File: storybook/stories/core/components/List/list-data.json (hunk 2 above)
Comment (ShachiniMekala) [reply storybook/stories/core/components/List/list-data.json:221]: Added

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+2024-12-10 - 7f771ad53e40a51c4394ffa76812cb062d9226c7 - Make select disable of non-deletable items in oxd-list
```
Comment (Super-Chama) [inline changelog.md]: Please follow the standard format

```[date] - [commit hash] - [relative path] - [change]```

File: components/src/core/components/ListTable/ListTable.vue [hunk 4]
```diff
@@ -291,6 +295,10 @@ export default defineComponent({
       return props.items;
     });
 
+    const allSelectDisabled = computed(() => {
```
Comment (Super-Chama) [inline components/src/core/components/ListTable/ListTable.vue:298]: this behaviour seems not implemented for cardtable and old list component. please check

Comment (ShachiniMekala) [reply components/src/core/components/ListTable/ListTable.vue:298]: Added it to the card table as well

**Review by Super-Chama — APPROVED** [review]

**Pull request merged**